tests/integration/for.spec.js

```diff
@@ -0,0 +1,24 @@
+/* global describe, it, cy, expect */
+
+describe('x-for directives', () => {
+  it('should not add duplicate items when navigating away and back to the page', () => {
+    cy.visit('http://127.0.0.1:8080/tests/res/for/index.html')
```

Comment:
we could use a `baseURL` and then could do `visit(/tests/res/for/index.html)`

Reply:
I've changed it but the assertion still need the full url so it hasn't improved drastically.